profiles/bicycle.lua

```diff
@@ -544,6 +544,7 @@ function turn_function(turn)
      turn.duration = turn.duration + profile.traffic_light_penalty
   end
   if properties.weight_name == 'cyclability' then
+      turn.weight = turn.duration
```

Comment:
There is no fallback here since the weight is assigned a few lines below. The main problem is that the turn penalty was only used for the duration, but we didn't also use it as basis for the turn weight (effectively disabling it).